examples/sklearnex/n_jobs.py

```diff
@@ -14,13 +14,8 @@
 # limitations under the License.
 # ==============================================================================
 
-# sklearnex doesn't have interface for threading configuration and not following
-# scikit-learn n_jobs yet. Thus it's requered to use daal4py package to set this.
-# nthreads parameter define number of threads used by sklearnex.
-# Without this code sklearnex would be using all system cores
-import daal4py
-
-daal4py.daalinit(nthreads=2)
+# sklearnex support `n_jobs` parameter for all patched estimators
+# even if original sklearn estimator doesn't
```

Comment:
I'd leave a bit modified sentence from the previous version of the code:

"n_jobs parameter defines the number of threads used by sklearnex.
Without this code sklearnex would be using all system cores."

Reply:
I mean in addition to the comment you've added.